tools/ci_build/github/apple/package_assembly_utils.py

```diff
@@ -70,6 +71,25 @@ def replace_template_variable(match):
         output.write(content)
 
 
+def filter_files(file_list, exclude_list):
```

Comment:
please add type annotations

Reply:
how about more explicit parameter names like `all_file_patterns` and `excluded_file_patterns`?